diff --git a/profiles/minimal/minimal.info b/profiles/minimal/minimal.info
index 6ec8f74..3e60bca 100644
--- a/profiles/minimal/minimal.info
+++ b/profiles/minimal/minimal.info
@@ -4,3 +4,5 @@ version = VERSION
 core = 7.x
 dependencies[] = block
 dependencies[] = dblog
+; Necessary for Forwardport doctrine (see http://drupal.org/node/1932290 )
+dependencies[] = moopapi
diff --git a/profiles/standard/standard.info b/profiles/standard/standard.info
index 3a65cfb..786da0a 100644
--- a/profiles/standard/standard.info
+++ b/profiles/standard/standard.info
@@ -23,3 +23,5 @@ dependencies[] = overlay
 dependencies[] = field_ui
 dependencies[] = file
 dependencies[] = rdf
+; Necessary for Forwardport doctrine (see http://drupal.org/node/1932290 )
+dependencies[] = moopapi
